src/org/thoughtcrime/securesms/ConversationActivity.java

```diff
@@ -554,9 +561,17 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
   @Override
   public void onBackPressed() {
-    Log.w(TAG, "onBackPressed()");
-    if (container.isInputOpen()) container.hideCurrentInput(composeText);
-    else                         super.onBackPressed();
+    if (container.isInputOpen()) {
+      container.hideCurrentInput(composeText);
+    }
+    else if (isSearchMode) {
```

Comment:
else if should be on the same line as  "{"
https://github.com/signalapp/Signal-Android/blob/0a569676f7a57144374a24faef566b2ca3233290/src/org/thoughtcrime/securesms/util/ExpirationUtil.java#L11